packages/xo-server/bin/xo-server

```diff
@@ -16,16 +20,6 @@ process.on('unhandledRejection', function (reason) {
   console.warn('[Warn] Possibly unhandled rejection:', reason && reason.stack || reason)
 })
 
-;(function (EE) {
-  var proto = EE.prototype
-  var emit = proto.emit
-  proto.emit = function patchedError (event, error) {
-    if (event === 'error' && !this.listenerCount(event)) {
-      return console.warn('[Warn] Unhandled error event:', error && error.stack || error)
-    }
-
-    return emit.apply(this, arguments)
-  }
-})(require('events').EventEmitter)
+catchGlobalErrors(log)
```

Comment:
You can also remove the code above, unhandled rejections are also handled by `log`.